modules/gridmap/doc_classes/GridMap.xml

```diff
@@ -69,6 +69,13 @@
 				The orientation of the cell at the given grid coordinates. [code]-1[/code] is returned if the cell is empty.
 			</description>
 		</method>
+		<method name="get_cell_neighbors" qualifiers="const">
+			<return type="Vector3i[]" />
+			<param index="0" name="cell" type="Vector3i" />
+			<description>
+				Returns an array of [Vector3i] map coordinates that neighbor the cell at coordinates [param cell], including unused cells.  Use [method get_cell_item] to check if the cell is used.
```

Comment:
```suggestion
				Returns an array of [Vector3i] map coordinates that are next to the cell at coordinates [param cell], including empty cells. Use [method get_cell_item] to check if a cell is not empty.
```
Alternatively, I'm not sure about the mention of `get_cell_item`. Feels unnecessary.
```suggestion
				Returns an array of [Vector3i] map coordinates that are next to the cell at coordinates [param cell], including empty cells.
```